Dockerfile

```
@@ -34,4 +33,4 @@ COPY . $APP_HOME
 
 EXPOSE 8000
 
-CMD /bin/sh -c "wait-for-it $DATABASE_HOST:${DATABASE_PORT:-5432} -- poetry run python manage.py migrate && poetry run uwsgi"
+CMD /bin/sh -c "wait-for-it $DATABASE_HOST:${DATABASE_PORT:-5432} -- poetry run python manage.py migrate && poetry run gunicorn --bind :8000 caluma.wsgi"
```

```suggestion
CMD /bin/sh -c "wait-for-it $DATABASE_HOST:${DATABASE_PORT:-5432} -- poetry run python manage.py migrate && poetry run gunicorn --workers 10 --access-logfile - --limit-request-line 16384 --bind :8000 caluma.wsgi"
```
Let's spin up at least 10 workers. The old config had 16, but [we did the same in mySAGW](https://github.com/adfinis/mySAGW/commit/7f5c73d50b6f428d1663523a3ab1055271b6f136)

Further we should increase the request size limit, in case we have large listing in the query string (we did that on all our caluma projects).